pom.xml

```diff
@@ -1,20 +1,27 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<parent>
+		<groupId>org.jenkins-ci.plugins</groupId>
 		<artifactId>plugin</artifactId>
-		<groupId>org.jvnet.hudson.plugins</groupId>
-		<version>1.357</version>
-		<relativePath>../pom.xml</relativePath>
+		<version>4.12</version>
 	</parent>
 
 	<groupId>org.jvnet.hudson.plugins</groupId>
 	<artifactId>postbuild-task</artifactId>
 	<packaging>hpi</packaging>
-	<name>Hudson Post build task</name>
-	<version>1.9-SNAPSHOT</version>
+	<name>Post build task</name>
+	<version>${revision}${changelist}</version>
 
-	<description>This plugin allows the user to execute a shell/batch task depending on the build log output.Java regular expression are allowed. If the string matches anywhere in the build log output, the script will execute.</description>
-	<url>http://wiki.hudson-ci.org/display/HUDSON/Post+build+task</url>
+	<properties>
+		<revision>1.9</revision>
+		<changelist>-SNAPSHOT</changelist>
+		<gitHubRepo>jenkinsci/postbuild-task-plugin</gitHubRepo>
+		<jenkins.version>2.204.6</jenkins.version>
+		<java.level>8</java.level>
+	</properties>
+	
+	<description>Allows the user to execute a shell/batch task depending on the build log output. Java regular expression are allowed. If the string matches anywhere in the build log output, the script will execute.</description>
```

Comment:
Might be a bit long for UI

Reply:
this doesn't get shown in the UI, index.jelly does https://github.com/jenkinsci/postbuild-task-plugin/blob/5d49bd8cfcbac24134fa09fbfeeda70ed1604e70/src/main/resources/index.jelly#L28